spec/unit/plugins/os_spec.rb

```diff
@@ -69,6 +69,36 @@
     end
   end
 
+  describe "on AIX" do
```

Comment:
We need the copyright in the spec as well

Copyright:: Copyright (c) 2014 Richard Manyanza.